app/assets/stylesheets/pages/settings.scss

```diff
@@ -110,3 +110,27 @@ $sidebar-item-highlight: #00c081;
 .ldap-conn-message-valid {
   color: $brand-success;
 }
+
+.cert-table {
+  text-indent: 10px;
```

Comment:
You can remove this because we are going to use `word-break` and `width` that will align stuff better.

Reply:
It's not clear to me what needs to be removed.  We use ".cert-table" below.  Are you referring to the "text-indent"?

Reply:
Done